src/idls/dab_registries/canister_registry.did.ts

```diff
@@ -0,0 +1,50 @@
+export default ({ IDL }) => {
+  const detail_value = IDL.Rec();
+  detail_value.fill(
+    IDL.Variant({
+      'I64' : IDL.Int64,
+      'U64' : IDL.Nat64,
+      'Vec' : IDL.Vec(detail_value),
+      'Slice' : IDL.Vec(IDL.Nat8),
+      'Text' : IDL.Text,
+      'True' : IDL.Null,
+      'False' : IDL.Null,
+      'Float' : IDL.Float64,
+      'Principal' : IDL.Principal,
+    })
+  );
+  const canister_metadata = IDL.Record({
+    'thumbnail' : IDL.Text,
+    'name' : IDL.Text,
+    'frontend' : IDL.Opt(IDL.Text),
+    'description' : IDL.Text,
+    'principal_id' : IDL.Principal,
+    'details' : IDL.Vec(IDL.Tuple(IDL.Text, detail_value)),
+  });
+  const operation_error = IDL.Variant({
+    'NotAuthorized' : IDL.Null,
+    'BadParameters' : IDL.Null,
+    'Unknown' : IDL.Text,
+    'NonExistentItem' : IDL.Null,
+  });
+  const operation_response = IDL.Variant({
+    'Ok' : IDL.Opt(IDL.Text),
+    'Err' : operation_error,
+  });
+  return IDL.Service({
+    'add' : IDL.Func(
```

Comment:
I think we should remove the admin-only methods for dab registries. No one can call these.

Reply:
I'm not sure I can do that but I'll try

Reply:
I can remove them from the IDL interface but I cannot remove it from the classes themselves. Maybe I can override the method to fail but it'd be the same as just allowing them to call it as it would fail either way

Reply:
okay, then I don't see a reason to insist on removing them. No one other than us can call them anyways.